app/src/main/java/com/bignerdranch/android/geoquiz/Question.kt

```diff
@@ -2,4 +2,6 @@ package com.bignerdranch.android.geoquiz
 
 import androidx.annotation.StringRes
 
-data class Question(@StringRes val textResId: Int, val answer: Boolean)
\ No newline at end of file
+data class Question(@StringRes val textResId: Int,
+                    val answer: Boolean,
+                    var isAnswered: Boolean = false)
```

Comment:
You have wrong indentation here. If you wanna have the fields from new line each, then be consistent for all, i.e start `textResId` from newline too.